compiler/modules/front/codeGen.bal

```diff
@@ -983,16 +988,28 @@ function codeGenCompoundAssignToVar(CodeGenContext cx,
                                     s:Expr rexpr,
                                     s:BinaryArithmeticOp|s:BinaryBitwiseOp op,
                                     Position pos) returns CodeGenError|StmtEffect {
-    s:Expr expr;
-    Position startPos = rexpr.startPos;
-    Position endPos = rexpr.endPos;
+    var [unnarrowedReg, assignments] = check assignmentRegister(cx, env, lValue.name, pos);
```

Comment:
I would expect this to look more like the code for codeGenCompoundAssignToListMember/MappingMember.

Reply:
fixed